### ISSUE TITLE
Burning fee allowed for system parachain?

### ISSUE THREAD

**Issue by xlc** [paritytech/cumulus#2320]
https://github.com/paritytech/cumulus/blob/66887505e6c5ef89ce9f88fc4ec368eb858b1b92/parachains/runtimes/assets/statemint/src/xcm_config.rs#L314

To my understanding, this will burn all XCM fees.

Two questions:

- Shouldn't fee be sent to collator?
- Can system parachain allowed to burn relaychain tokens (without notifying relaychain)? Wouldn't this cause imbalances? i.e. balance mismatch between relaychain's checking account and parachain's total issuance.

**Comment by joepetrowski** [comment]
Yes, I agree

**Comment by gavofyork** [comment]
Yes, when `FeeManager` is used, they would be better sent to a proper account.
